db/seeds/base/add_users.rb

```diff
@@ -0,0 +1,53 @@
+otp_testing_code = "00000"
+
+# Create admin user
+Admin.create!(
+  email: "admin@example.com",
+  full_name: "example admin",
+  otp_hash: otp_testing_code,
+  otp_expires_at: "3000-01-01 00:00:00.000000000 +0000",
+  super_admin: false,
+)
+
+# Create admin user
+Admin.create!(
+  email: "superadmin@example.com",
+  full_name: "example super admin",
+  otp_hash: otp_testing_code,
+  otp_expires_at: "3000-01-01 00:00:00.000000000 +0000",
+  super_admin: true,
+)
+
+@single_app_user = User.create!(
```

Comment:
I wonder if we can avoid creating the users here and instead create them with the applications to encapsulate the logic in that file instead? I think this way we avoid linking across files also we avoid the risk of having 1 user attached to too many applications?